pkg/containerspec/containerspec.go

```diff
@@ -34,9 +35,10 @@ func (n KubeResourceQuantity) MarshalJSON() ([]byte, error) {
 
 // PaastaContainerSpec : Spec for any paasta container with basic fields and utilities
 type PaastaContainerSpec struct {
-	CPU    *KubeResourceQuantity `json:"cpus"`
-	Memory *KubeResourceQuantity `json:"mem"`
-	Disk   *KubeResourceQuantity `json:"disk"`
+	CPU       *KubeResourceQuantity `json:"cpus"`
+	Memory    *KubeResourceQuantity `json:"mem"`
+	Disk      *KubeResourceQuantity `json:"disk"`
+	DiskLimit *KubeResourceQuantity `json:"disk_limit"`
```

Comment:
As I mentioned earlier, I think it would be better to have this marked as optional and omitempty. I would really like to avoid people to have to specify this property in yelpsoa-configs.

Reply:
I am talking about `DiskLimit`. GitHub decided to show the whole block in the diff

Reply:
I don't think there is any problem with the behavior you codified, if not that you have another comment about that from @vkhromov